lib/pbench/server/api/resources/query_apis/datasets_detail.py

```diff
@@ -69,15 +68,15 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
             requested set of metadata to the postprocess method.
         """
         user = json_data.get("user")
-        name = json_data.get("name")
+        dataset_name = json_data.get("dataset_name")
         start = json_data.get("start")
         end = json_data.get("end")
```

Comment:
I definitely do like the idea of converting all of these to use SQL validation and the index map; we'll drop three of these four primary parameters!